### DISCUSSION TITLE
Controlling TF-series mixers directly from within Qlab

### DISCUSSION THREAD

**Discussion by snurrgnu** [BrenekH/yamaha-rcp-docs#2] [General]
Hello,

Found this and have tested it from Qlab.
Works like a charm.
But working in Qlab, I have to use applescript to use a pythonscript to communicate with the TF1, so I asked a friend who's a coder for some help to try and communicate directly from applescript within Qlab and he came up with the following solution that I would like to share with others.

He used NetCat to send the commands from within Qlab.

tell application id “com.figure53.Qlab.4” to tell front workspace
       **do shell script “echo \”set MIXER:Current/InCh/Fader/on 7 0 0\” | nc 192.168.1.100 49280”**
end tell

But when I sent this to Qlab they replied that the first and last lines could be obmitted so the end result would look like this.

 **do shell script “echo \”set MIXER:Current/InCh/Fader/on 7 0 0\” | nc 192.168.1.100 49280”**

I would like to link to this page from the Qlab discussion board so that others can find the commands that you've gathered.
Would that be OK?

**Comment by BrenekH** [comment]
Absolutely! The more people who know about this stuff the better!
